cypress/support/login.js

```diff
@@ -11,13 +12,18 @@ Cypress.Commands.add('login', (username, password) => {
 
   cy.visit('/');
 
-  cy.do([
-    TextField('Username').fillIn(username),
-    TextField('Password').fillIn(password),
-    Button('Log in').click(),
-  ]);
+  // Todo: find the way to wrap interactor to cy chainable object
+  cy.get('input[name=username]', getLongDelay()).should('be.visible');
+  cy.do(TextField('Username').fillIn(username));
+  cy.do(TextField('Password').fillIn(password));
+  cy.do(Button('Log in').click());
+
+  // TODO: find the way how customize waiter timeout in case of interactors(cy.wrap may be)
```

Comment:
_defaultCommandTimeout_ in _cypress.json_ help us with it. If it need we can set new value of this parametr. Why we need new ways for expecting elements?

Reply:
if we would say about regular situations - yes, we can work through defaultCommandTimeout. From other side, some set actions can be complicated and related with long responses from application(for example, it's required in some test to create object with maximum volume of test data). For such cases I suggest to specify additional delay through getLongDelay(), which will additionally indicate to us such special places in tests and fragments.

Reply:
ok, I understood, thanks